src/js/components/Grommet/__tests__/__snapshots__/Grommet-test.js.snap

```diff
@@ -265,7 +265,7 @@ exports[`Grommet ssr rendering Mozilla/5.0 (iPad; CPU OS 1`] = `
   <h1
     className="c1"
   >
-    Received size medium for Mozilla/5.0 (iPad; CPU OS 11_2_1 like Mac OS X) AppleWebKit/604.4.7 (KHTML, like Gecko) Mobile/15C153 [FBAN/FBIOS;FBAV/156.0.0.41.97;FBBV/89172188;FBDV/iPad5,3;FBMD/iPad;FBSN/iOS;FBSV/11.2.1;FBSS/2;FBCR/;FBID/tablet;FBLC/en_GB;FBOP/5;FBRV/0]
+    Received size large for Mozilla/5.0 (iPad; CPU OS 11_2_1 like Mac OS X) AppleWebKit/604.4.7 (KHTML, like Gecko) Mobile/15C153 [FBAN/FBIOS;FBAV/156.0.0.41.97;FBBV/89172188;FBDV/iPad5,3;FBMD/iPad;FBSN/iOS;FBSV/11.2.1;FBSS/2;FBCR/;FBID/tablet;FBLC/en_GB;FBOP/5;FBRV/0]
```

Comment:
Should these have changed?

Reply:
I agree it doesn't look right for the iPad to receive the `large` vs `medium` breakpoint. I'll have to take a deeper look into the SSR role and its impact on the logic. 
@oorestisime, any insights could be helpful here.